src/utils/validation.js

```diff
@@ -18,3 +19,21 @@ export const nameValidator = (name) => {
 
 export const emailValidator = (email) =>
   validator.isEmail(email) ? null : EMAIL_ERROR;
+export const passwordValidator = (password) => {
+  if (validator.isEmpty(password)) {
+    return PASSWORD_EMPTY_ERROR;
+  }
+  // TODO: check strength of password
+  // if (
+  //   !validator.isStrongPassword(password, {
+  //     minLength: 8,
+  //     minLowercase: 1,
+  //     minUppercase: 1,
+  //     minNumbers: 1,
+  //     minSymbols: 1,
+  //   })
+  // ) {
+  //   return PASSWORD_ERROR;
+  // }
```

Comment:
actually it doesn't make sense to check the strength here since it's already saved in db right? Since we decided we will only define the password in the profile, no need to check, but it's good to remember to do it when you'll add the feature in compose!

Reply:
you are right. I found this during the research I did and left it here so I don't forget it. I remove it and add it to the compose